classes/Bili/Date.php

```diff
@@ -272,9 +299,10 @@ public static function fixShortYearInDate($strDate)
      * @param string $strOutFormat
      * @return string
      */
-    public static function convertDate($strDate, $strInFormat, $strOutFormat)
+    public static function convertDate(string $strDate, string $strInFormat, string $strOutFormat): string
```

Comment:
Please rename `$strInFormat` to `$strInIsoFormat` and `$strOutFormat` to `$strOutIsoFormat`. Also adjust the doc block.